src/OrbitGl/MajorPagefaultTrack.cpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "MajorPagefaultTrack.h"
+
+#include <absl/strings/substitute.h>
+
+namespace orbit_gl {
+MajorPagefaultTrack::MajorPagefaultTrack(Track* parent, TimeGraph* time_graph,
+                                         orbit_gl::Viewport* viewport, TimeGraphLayout* layout,
+                                         const std::string& cgroup_name,
+                                         const orbit_client_model::CaptureData* capture_data,
+                                         uint32_t indentation_level)
+    : BasicPagefaultTrack(parent, time_graph, viewport, layout, "Major Pagefault Track",
+                          cgroup_name, capture_data, indentation_level) {
+  index_of_series_to_highlight_ = static_cast<size_t>(SeriesIndex::kProcess);
+}
+
+std::string MajorPagefaultTrack::GetTooltip() const {
+  return "Shows major pagefault statistics. A major pagefault occurs when the requested page does "
```

Comment:
Should this tooltip be splitted in multiple lines? I can't see it in the screenshot

Reply:
https://screenshot.googleplex.com/692dnUqA8FRUb8U I was a single line previously. Now I changed to two lines.